README.md

```diff
@@ -87,91 +108,80 @@ echo -e "admin\n${CONCOURSE_ATC_PASSWORD}" | \
    ${FLY_CMD} -t ${DEPLOY_ENV} login -k -c "https://${DEPLOY_ENV}-concourse.cf.paas.alphagov.co.uk"
 ```
```

Comment:
Do we need to explain how to get the fly command? ... and setting the FLY_CMD variable isn't covered in the readme (although arguably that is pretty obvious if you know what you're doing)